src/context/UserContext.js

```diff
@@ -3,9 +3,22 @@ import { createContext, useContext, useMemo, useState } from 'react';
 export const UserContext = createContext();
 
 const UserProvider = ({ children }) => {
-  const [user, setUser] = useState('');
+  const [user, setUser] = useState(null);
+  console.log(process.env.AUTH_EMAIL);
```

Comment:
We should probably get rid of this console.log, it's also exposing the env variables